tests-wtr/rules/rule-compaction.test.ts

```diff
@@ -11,7 +11,7 @@ describe('RuleCompaction', () => {
         const encoded = encodeRules(rules, null);
         const decoded = decodeRules(encoded);
         expect(rules.length).to.equal(decoded.length);
-        const ignoredKeys = ['comment', 'optIn', 'vendorUrl', '_metadata'];
+        const ignoredKeys = ['comment', 'optIn', 'vendorUrl', '_metadata', 'detectPopup', 'detectCmp', 'optOut', 'test'];
```

Comment:
Nit: it's confusing to see all these in "ignored" steps, I wonder if there's a cleaner way to ignore `comment`s, we could just deep-remove this key from the whole json object.
Feel free to ignore though, it's just a test